content/docs/guides/neon-authorize.md

```diff
@@ -59,21 +70,18 @@ In a traditional setup, you might handle authorization for a function directly i
 ```typescript shouldWrap
 export async function insertTodo(newTodo: { newTodo: string; userId: string }) {
   const { userId } = auth(); // Gets the user's ID from the JWT or session
```

Comment:
Since we already have `auth()` here, let's get the `getToken` from it:

```.tsx
const { userId, getToken } = auth();
// also added the await here
const authToken = await getToken();
```